packages/@n8n/codemirror-lang/.eslintrc.cjs

```diff
@@ -0,0 +1,14 @@
+const sharedOptions = require('@n8n_io/eslint-config/shared');
+
+/**
+ * @type {import('@types/eslint').ESLint.ConfigData}
+ */
+module.exports = {
+	extends: ['@n8n_io/eslint-config/frontend'],
```

Comment:
`@n8n/codemirror-lang` is a FE package but cannot benefit from Vue rulesets. Maybe `eslint-config/base`?